FE/src/components/AccountMenus/UserProfile/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { FaRegUser } from "react-icons/fa";
+import { FiLogOut } from "react-icons/fi";
+import { useAppContext } from "../../../context.js";
+import { UserImg } from "../LoginMenu/styled-components";
+import { StatsHolder, UserProfileContainer, VerticalLine } from "./styles.js";
+
+function getUserImage(image) {
+	return image === null ? <FaRegUser size="80px" /> : <img src={image} alt="User chosen profile" />;
```

Comment:
Por defeito o argumento size do icon é em pixels:
```suggestion
	return image === null ? <FaRegUser size={80} /> : <img src={image} alt="User chosen profile" />;
```